routes/web.php

```diff
@@ -381,19 +381,19 @@
                 [EventAttendeesController::class, 'postMessageAttendees']
             )->name('postMessageAttendees');
 
-            Route::get('{event_id}/attendees/single_message',
+            Route::get('{attendee_id}/attendees/single_message',
```

Comment:
These have regressed from the laravel 6.5 upgrade. The controller methods require an `attendee_id` for each of these routes instead of an event and I think these happened because of copy paste. I tested them again and they seem to work fine